### DISCUSSION TITLE
part-ii/2016-114/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#421] [Question comments]
Part II 2016 Principles of Statistics: Paper 2, Section II, 

https://questions.tripos.org/part-ii/2016-114/

**Comment by ArthurConmy** [comment]
<details>
<summary>b)</summary>

Compute (tis painful) $\mathbb{ E}[S^2] = \theta$ hence the result follows from Cramer-Rao as the estimator is unbiased.
</details> 

<details>
<summary>c)</summary>

Adjust Cramer-Rao proof to show that $ \text{Var}( \tilde{\theta}_n ) \ge \frac{(1+B'(\theta))^2}{nI(\theta)} \ge \frac{1}{n} $ hence as MSE = Bias^2 + Variance, the result follows. 
</details>
